res/values/setup.xml

```diff
@@ -26,17 +26,6 @@
     <string name = "send_files_to_other_apps">on</string>
     <string name = "share_feature">on</string>
     
-    
-    <!-- Colors -->
-    <color name="login_background_color">#FFFFFF</color>
```

Comment:
This is not possible right now. These colours need to be kept in this file, with the same names.